lib/rspec/support/ruby_features.rb

```diff
@@ -90,6 +90,23 @@ def ripper_supported?
         end
       end
 
+      begin
+        verbose_was = $VERBOSE
+        $VERBOSE = nil
+
+        String::Hash
```

Comment:
This is failing on windows because of the void context of this lookup. I think assigning this to a local that gets thrown away would solve it 👍